rasa/cli/arguments/train.py

```diff
@@ -101,6 +102,30 @@ def add_compare_params(
     )
 
 
+def add_dry_run_param(
+    parser: Union[argparse.ArgumentParser, argparse._ActionsContainer]
+) -> None:
+    """Add `--dry-run` argument to a specified `parser`.
```

Comment:
Because we agreed to use the descriptive style
```suggestion
    """Adds `--dry-run` argument to a specified `parser`.
```